### ISSUE TITLE
Without layout argument, ordered_transitions is required or nothing will be displayed

### ISSUE THREAD

**Issue by translunar** [pytransitions/transitions-gui#28]
With this very simple minimum working example, nothing is displayed unless I add the `layout` argument.

```
from transitions_gui import WebMachine
import time

states = ['locked', 'auto', 'spray', 'overflow']
# initializing the machine will also start the server (default port is 8080)
machine = WebMachine(states=states, initial='locked', name="Char Quencher",
                     ignore_invalid_triggers=True,
                     auto_transitions=False)

machine.add_transition('unlock', 'locked', 'auto') # locked -> auto (user commandded)
machine.add_transition('nominal_temp', ['spray', 'auto'], 'auto') # spray -> auto (pump stops)
machine.add_transition('lock', '*', 'locked') # all -> locked (user commanded)
machine.add_transition('flood', 'spray', 'overflow') # spray lasts too long, stick it in timeout
machine.add_transition('unlock', 'overflow', 'auto') # allow user to override timeout (user commanded)
machine.add_transition('high_temp', 'auto',  'spray') # auto -> spray (pump starts)
machine.add_transition('high_temp', 'spray', '=') # keep spraying while temp is high
try:
    while True:
        time.sleep(5)
        #machine.next_state()
except KeyboardInterrupt:  # Ctrl + C will shutdown the machine
    machine.stop_server()
```

This also means I have to reset the layout every time I load, because I can't save a layout and load from that one.

Without layout argument:
<img width="811" alt="image" src="https://user-images.githubusercontent.com/165284/199802849-5edf623c-8935-458e-9756-f1395b8a2bd1.png">

With layout argument:
<img width="740" alt="image" src="https://user-images.githubusercontent.com/165284/199802955-551d666a-f17f-4955-bed2-a1f47a8f184d.png">

I just installed transitions and transitions-gui a day or two ago using pip, so seems like it's the current version.

Interestingly, the basic example given in Quickstart in the readme *does* in fact work without setting a layout. It seems like the transitions need to be in place before the machine finishes construction.

Is there a way to add transitions post-construction with WebMachine?

**Comment by aleneum** [comment]
Hello @translunar,

which browser are you using? I just tested the code example above with PyPI version and GitHub version of `transitions-gui` and the result looks fine, at least with (MacOS) Firefox and Chrome:

<img width="745" alt="Screenshot 2022-11-05 at 13 47 41" src="https://user-images.githubusercontent.com/205986/200120635-d2bd3e48-6ff4-4fb9-a8b0-9374042dbe69.png">

**Comment by aleneum** [comment]
Checked again after dependency update. I will close this issue because it went stale a while ago. Feel free to reopen a new issue if this is still relevant for you.
